site/app/controllers/admin/AdminGradeableController.php

```diff
@@ -246,7 +248,7 @@ public function getGradeableJson(Gradeable $gradeable): array {
             $return_json['bulk_upload'] = $gradeable->isBulkUpload();
             if ($gradeable->isTeamAssignment()) {
                 $team_properties = [
-                    'team_max_size' => $gradeable->getTeamSizeMax(),
+                    'team_size_max' => $gradeable->getTeamSizeMax(),
```

Comment:
Does this break backwards compatibility with existing JSONs instructors may have?

Reply:
The UploadJSON function expects the variable to be `team_size_max`, so yes it is not 'backwards compatible', but the old JSON's wouldn't have the valid variable either.